src/particles/elements/Sbend.H

```diff
@@ -54,23 +54,38 @@ namespace impactx
             amrex::ParticleReal const y = p.pos(1);
             amrex::ParticleReal const t = p.pos(2);
 
+            // initialize output values of momenta
+            amrex::ParticleReal pxout = px;
+            amrex::ParticleReal pyout = py;
+            amrex::ParticleReal ptout = pt;
```

Comment:
I think this is only needed for `px`, not for `py` and `pt` in the sbend.

Reply:
Yes, that's true.  See the previous comment.